satellite/metainfo/metainfo.go

```diff
@@ -247,7 +247,28 @@ func (endpoint *Endpoint) CommitSegment(ctx context.Context, req *pb.SegmentComm
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
+	exceeded, limit, err := endpoint.projectUsage.ExceedsStorageUsage(ctx, keyInfo.ProjectID)
```

Comment:
I think this will race logically.

Reply:
As agreed, that this is fine for now until we have the respective RPC Changes in